examples/download_and_train.py

```diff
@@ -93,7 +96,7 @@
 with open(prefs_save_path, "w") as fp:
     json.dump(prefs, fp)
 
-command_str = "fnet train {} --gpu_ids {}".format(prefs_save_path, gpu_id)
+command_str = "fnet train --json {} --gpu_ids {}".format(prefs_save_path, gpu_id)
```

Comment:
fstrings are cool / I think they are even more legible. 🤷‍♀ 

```suggestion
command_str = f"fnet train --json {prefs_save_path} --gpu_ids {gpu_id}"
```